random_menu.rb

```diff
@@ -0,0 +1,19 @@
+# Random Menu Generator for Boozy Desserts (Lunch) - Loraine Kanervisto
```

Comment:
Hi, Loraine! Before I begin: one of my drinks was "frosty champagne jello" :cocktail: